lib/commons/dom/get-visible-child-text-rects.js

```diff
@@ -0,0 +1,80 @@
+import { getNodeFromTree, memoize } from '../../core/utils';
+import { sanitize } from '../text';
+import { getRectCenter, isPointInRect, getIntersectionRect } from '../math';
+import getOverflowHiddenAncestors from './get-overflow-hidden-ancestors';
+
+/**
+ * Get the visible text client rects of a node.
+ * @method getVisibleChildTextRects
+ * @memberof axe.commons.dom
+ * @instance
+ * @param {Element} node
+ */
+const getVisibleChildTextRects = memoize(
+  function getVisibleChildTextRectsMemoized(node) {
+    const vNode = getNodeFromTree(node);
+    const nodeRect = vNode.boundingClientRect;
+    const clientRects = [];
+    const overflowHiddenNodes = getOverflowHiddenAncestors(vNode);
+
+    Array.from(node.childNodes).forEach(elm => {
```

Comment:
Nitpick; this doesn't need Array.from. NodeList has a forEach method. Or you could use `for ... of`.